libdrgn/python/stack_trace.c

```diff
@@ -5,6 +5,19 @@
 #include "../stack_trace.h"
 #include "../util.h"
 
+StackTrace *StackTrace_wrap(struct drgn_stack_trace *trace) {
```

Comment:
This one doesn't lead to any awkward casts, but just for consistency with the other `_wrap` functions maybe it should return a `PyObject *`, too.